.one-pipeline.yaml

```diff
@@ -460,6 +456,13 @@ acceptance-test:
       exit 0
     fi
 
+    SKIP_ACCEPTANCE_TEST=$(get_env SKIP_ACCEPTANCE_TEST)
```

Comment:
the cluster request and config (in this stage as well as in `setup` stage) can be skipped since the cluster is only needed for acceptance tests. Same is true for PERIODIC_SCAN. If either one is set, cluster request and config can be skipped.